nav2_regulated_pure_pursuit_controller/include/nav2_regulated_pure_pursuit_controller/regulated_pure_pursuit_controller.hpp

```diff
@@ -234,6 +239,13 @@ class RegulatedPurePursuitController : public nav2_core::Controller
    */
   double findDirectionChange(const geometry_msgs::msg::PoseStamped & pose);
 
+  /**
+   * @brief Normalizes angle in range [-pi, pi]
+   * @param angle Angle to normalize
+   * @return Normalized angle
+   */
+  double angleNormalize(double angle);
```

Comment:
Pass by const reference

Reply:
Replaced with `ros/angles`, as pointed out in the other comment.